README.md

```diff
@@ -189,15 +160,76 @@ Example request URI:
 | ```NO_CONTENT(204)```                   | ```Data successfully deleted from database``` |
 | ```FORBIDDEN(403)```                    | ```Auth Internal IDs do not match```          |
 
+#### POST /register-trust and POST /register-ua
+
+---
+Submits a registration request to the downstream Register API. This API is feature switched behind
+the `Use stub for submissions to Registration` switch, so it can be stubbed using the Register test endpoint described
+below.
+
+##### Request:
+
+Body:
+
+Trust:
+
+```
+{
+"trust": {
+     "sautr": 1234567890,
+     "regime": "VATC"
+     }
+}
+```
+
+Unincorporated Association:
+
+```
+{
```

Comment:
Guess we need to add "unincorporatedAssociation" as json object

Reply:
Changed it so that we don't need that in the request body to our route, updated the Trust route too but had missed it in the ReadMe so should be fixed now

Reply:
thats right, we didnt need "trust" here that you have already got rid of it